modules/cells/src/main/java/dmg/cells/nucleus/SystemCell.java

```diff
@@ -356,15 +356,15 @@ public void uncaughtException(Thread t, Throwable e)
         Throwable root = Throwables.getRootCause(e);
         if (root instanceof FileNotFoundException) {
             if (root.getMessage().contains("Too many open files")) {
-                _log.error(AlarmMarkerFactory.getMarker(PredefinedAlarm.OUT_OF_FILE_DESCRIPTORS,
+                logger.error(AlarmMarkerFactory.getMarker(PredefinedAlarm.OUT_OF_FILE_DESCRIPTORS,
                                                         getCellDomainName(),
                                                         getCellName()),
-                           "Uncaught exception in thread " + t.getName(),
+                           "Uncaught exception in thread {}", t.getName(),
```

Comment:
Ok. in did, depending on backend logger implementation I get a different result:
```
14:46:02.149 [main] ERROR net.kofemann.coffeesnack.LoggingTest - This is multi args statement [one, two, three] with throwable {}
java.lang.Exception: fail with args
	at net.kofemann.coffeesnack.LoggingTest.main(LoggingTest.java:24)
14:46:02.153 [main] ERROR net.kofemann.coffeesnack.LoggingTest - This statement without args, but with a throwable {}
java.lang.Exception: fail alone....
	at net.kofemann.coffeesnack.LoggingTest.main(LoggingTest.java:28)
```
which is for me an indication, that we can't relay on it. We stick to javadoc, sorry.